README.md

```diff
@@ -43,6 +43,22 @@ This boilerplate comes with batteries included, you’ll find:
 4. Create a new Git repository (`git init`)
 5. Create the initial Git commit (`git commit -a -m "Initial commit"`)
 
+## Preferred libraries
+
+Some batteries aren’t included since all projects have their own needs and requirements. Here’s a list of our preferred libraries to help you get started:
+
+| Category                    | Libraries                                                                                           |
+| --------------------------- | --------------------------------------------------------------------------------------------------- |
+| Authentication              | [`ueberauth`](https://github.com/ueberauth/ueberauth), [`pow`](https://github.com/danschultzer/pow) |
+| Asynchronous job processing | [`oban`](https://github.com/sorentwo/oban)                                                          |
+| Emails                      | [`bamboo`](https://github.com/thoughtbot/bamboo), [`swoosh`](https://github.com/swoosh/swoosh)      |
+| File upload                 | [`waffle`](https://github.com/elixir-waffle/waffle)                                                 |
+| HTTP client                 | [`tesla`](https://github.com/teamon/tesla)                                                          |
+| HTML parsing                | [`floki`](https://github.com/philss/floki)                                                          |
+| Pagination                  | [`scrivener`](https://github.com/drewolson/scrivener)                                               |
+| Mocks                       | [`mox`](https://github.com/dashbitco/mox), [`mimic`](https://github.com/edgurgel/mimic)             |
+| Search                      | [`elasticsearch`](https://github.com/danielberkompas/elasticsearch-elixir)                          |
```

Comment:
Shouldn’t we link to Hex.pm packages instead of GitHub projects?

Reply:
I find the GitHub links more helpful but yeah, I can change it if that’s what we want to expose in the `README.md`.

Reply:
I like both. `Hex` is the distribution channel like `NPM` and does expose the `Github` link as well, but you rarely find what you are looking for in the `Hex` website, to be frank.

Reply:
It’s a small detail indeed, but I think since we’re referencing packages that we include in our `mix.exs`, we should use the "thing that gets pulled from when running mix deps.get" and that’s the hex.pm package 😄